src/Illuminate/Support/helpers.php

```diff
@@ -813,6 +813,25 @@ function str_slug($title, $separator = '-')
     }
 }
 
+
+if (! function_exists('stub')) {
```

Comment:
This must be defined in the foundation helpers because it depends on the base_path function that is only defined in there, not in "support".